### PR TITLE
Added missing npmignore

### PR DESCRIPTION
**Pull request by JGAntunes**
If I understood the dignified pipeline correctly I think you need a `.npmignore` file, as the lack of it implies that, by default, npm publish will use `.gitignore` which will leave out `/lib` path.

Basically I've copy pasted the .gitignore file, leaving out files npm already ignores by default and the `/lib` path.

### CODE REVIEW

**Comment by xicombd** [comment]
LGTM

**Comment by diasdavid** [comment]
🌟 @JGAntunes thank you for catching this. @dignifiedquire @nginnever looks like the other dignified PRs have to be changed :)

File: .npmignore [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+# Compiled binary addons (http://nodejs.org/api/addons.html)
+build/Release
+
+# Optional npm cache directory
+.npm
+
+# Optional REPL history
+.node_repl_history
+
+dist
```
Comment (dignifiedquire) [inline .npmignore]: `dist` should not be in here, we want that to be published as well!
cc @diasdavid

**Comment by JGAntunes** [comment]
@dignifiedquire @diasdavid sorry, didn't realise that. I can submit another PR to fix this if you think it's worth it.

**Comment by dignifiedquire** [comment]
@JGAntunes that would be great thanks

**Pull request merged**